FundacionParaguaya/MentorApp
# Feature/demo surveys

**Describe the changes**
Handling of demo surveys based on surveyConfig property **isDemo**

**To Test**
_Currently the demo surveys we have are "Stoplight Demo - English" and "Stoplight Demo - Spanish"_
1. Take one of the pointed above surveys.
2. In PrimaryParticipant and FamilyMembers screens , the form fields should be prefilled with random data.
3. Try submitting the demo survey at the end and see if it syncs.
4. Finally take a regular survey different from the demo surveys and make sure everything works as expected

This commit also solves #992
